lib/workers/repository/init/merge.ts

```diff
@@ -187,15 +188,20 @@ export async function mergeRenovateConfig(
     };
   }
   const configFileParsed = repoConfig?.configFileParsed || {};
+  const configFileAndEnv = await mergeStaticRepoEnvConfig(
+    configFileParsed,
+    process.env,
+  );
   if (is.nonEmptyArray(returnConfig.extends)) {
-    configFileParsed.extends = [
+    configFileAndEnv.extends = [
       ...returnConfig.extends,
-      ...(configFileParsed.extends || []),
+      // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
why this? it seems your local installation is not uptodate? run `pnpm i`?

Reply:
https://github.com/renovatebot/renovate/actions/runs/13413637678/job/37469179680
github ci failed :thinking:

Reply:
I wonder why we don't use `??` operator here should work because we already migrated this field to an array.

Reply:
tbh, i think `||` is safer in this case regardless.

Reply:
I don't think so. It's to be lazy. `configFileParsed.extends` should never be something different thang `null`, `undefined` or an `array` here. We should really get rid of such lazy safeguards.